Thorium-Sim/thorium-nova
# Drag select

## Description of Changes

**UI:**

- Store selected objects as a list of IDs instead of a single ID
- Add the useDragSelect hook.
- Apply the useDragSelect hook to the things happening in the starmap.

## Related Issue








Refs #283 

## How do you know the changes work correctly?






Manual testing:
- Start a flight
- Go to the flight director screen
- Open the starmap core
- Hold Shift and drag the mouse over the player contact
- It should select and deselect the player at the appropriate times.
